lib/views/login_register/password_reset_sent.dart

```diff
@@ -30,11 +29,9 @@ class PasswordResetSentScreen extends StatelessWidget {
               crossAxisAlignment: CrossAxisAlignment.start,
               children: <Widget>[
                 Text(
-                  'If your email is associated with an account, you will recived an email with a link to reset your password in your inbox.',
-                  style: TextStyle(
-                    color: ThcColors.darkBlue,
-                    fontSize: 20,
-                  ),
+                  'If your email is associated with an account, '
+                  'you will recived an email with a link to reset your password in your inbox.',
```

Comment:
Having 2 strings next to each other like this counts as a single string, and you don't have to deal with a super long line of text.